planet-kaggle-blog.md

```diff
@@ -0,0 +1,69 @@
+
+In the above figure, the ground truth tags (ie. tagged by hand) for the Planet Kaggle dataset are bolded. Green bolded tags are correct. Unbolded and uncolored tags mean that they have been incorrectly predicted for the chip. Red bolded tags are ones missed by the network prediction. As with many Kaggle competitions involving image identifcation, there are multiple stages in the prediction process that can be optimized for better accuracy. Improvements can be made in pre-processing the training data, switching or combining model architectures, adjusting optimizers, learning rates and augmenting the testing data. We placed 23rd overall out of nearly a thousand teams with a private leaderboard prediction accuracy of 93.154% using the following techniques.
+
+We normalized the provided images to be within a standardized range and used a range of image augmentations during the testing phase. Rotation, zooming in/out, flipping and cropping did increase the quality of prediction in some cases. We used a multi-architecture ensemble that took a majority vote over the predicted labels across 15 convolutional neural networks: (5) [ResNet50](https://arxiv.org/abs/1512.03385), (5) [Inception v3](https://arxiv.org/abs/1512.00567) and (5) [DenseNet121](https://arxiv.org/abs/1608.06993). These models were all trained using Adam as the optimizer, a cyclically decaying learning rate and binary cross-entropy as the loss function.
+
+The winning Kaggler, bestfitting, obtained a score of 93.318% accuracy with their solution. More details on this approach can be found in bestfitting's [solution summary](https://www.kaggle.com/c/planet-understanding-the-amazon-from-space/discussion/36809).
+
+A point of note: submissions to the competition were ranked using F2 score.  The F2 scoring metric tended to overlook mislabeling in underrepresented tags. Many high-performing trained their networks to improve the F2 score, we decided to not use this method. If we trained our model according to this metric, there would be less incentive for correctly identifying the rarer labels. Our method was conducted with the premise of fairly differentiating and labeling the uncommon tags. Often, the goal in using machine tagging is to teach the model to effectively and accurately identify interesting and anomalous features in a set of largely similar data.
+
+### Misconfigured Labels
+
+In order for this task to be performed accurately, it is crucial that the ground truth be genuinely truthful. In an ideal world, the criteria for a label is clear and distinct to humans and similarly obvious to trained neural networks. Unfortunately, Planet's crowd-labeled dataset had noticeable amounts of ambiguous and, even worse, clearly incorrect labels. For example, we can see that the network often mistakes when to label a chip with the tag `agriculture`. However, if we examine the ground truth tags for each chip, it's not obvious that the human classifications are correct either.
```

Comment:
'In order for this task to be performed accurately, it is crucial that the ground truth be genuinely truthful.' The ground truth is reality, so it's always correct by definition. I think you mean to say something more like 'the human-provided labels are mostly correct.'